src/testHelpers/src/main/kotlin/org/vaccineimpact/orderlyweb/test_helpers/JSONValidator.kt

```diff
@@ -64,36 +61,25 @@ class JSONValidator
 
     private fun checkResultSchema(json: JsonNode, expectedStatus: String)
     {
-        assertValidates(responseSchema, json)
+        assertValidates("Response", json)
         val status = json["status"].textValue()
         Assertions.assertThat(status)
                 .isEqualTo(expectedStatus)
     }
 
-    private fun readSchema(name: String): JsonNode = JsonLoader.fromResource("/spec/$name.schema.json")
-
-    private fun assertValidates(schema: JsonNode, json: JsonNode)
+    private fun assertValidates(name: String, json: JsonNode)
     {
-        val report = schemaFactory.getJsonSchema(schema).validate(json)
+        val file = File("../../docs/spec/$name.schema.json").toURI()
+        val report = schemaFactory.getJsonSchema(file.toString()).validate(json)
         if (!report.isSuccess)
         {
-            Assertions.fail("JSON failed schema validation. Attempted to validate: $json against $schema. Report follows: $report")
+            Assertions.fail("JSON failed schema validation. Attempted to validate: $json against $name. Report follows: $report")
         }
     }
 
     private fun makeSchemaFactory(): JsonSchemaFactory
     {
-        val namespace = "resource:/spec/"
```

Comment:
I'm not sure why this was in here! Was it needed for the reporting api?

Reply:
No, they were being loaded from the java resources location (at a subdirectory called `/spec/`) - I'm now loading them from a file location specifying the file path instead.